testbackup/DetailPresenterTests.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DetailPresenterTests.swift
+//  NetworkingProjectTests
+//
+//  Created by Timofey Privalov on 11.10.2021.
+//
+
+@testable import NetworkingProject
+import XCTest
+
+final class DetailMovieMockView: DetailMovieViewProtocol {
+    func success() {}
+}
+
+final class DetailPresenterTest: XCTestCase {
+    var view: DetailMovieMockView!
+    var presenter: DetailMoviePresenterProtocol!
+    var movieAPIService: MovieAPIServiceProtocol!
+    var imageService: ImageServiceProtocol!
+    var datastorageService: DataStorageServiceProtocol!
+    var router: RouterProtocol!
+    var movieDetail: MovieDetail!
+
+    override func tearDown() {
+        view = nil
+        movieAPIService = nil
+        imageService = nil
+        datastorageService = nil
+        router = nil
+        presenter = nil
+    }
+
+    func testGetSuccesMovies() {
+        let movieDetail = MovieDetail(
+            backdropPath: "baz",
+            budget: 1,
+            id: 12,
+            overview: "foo",
+            originalTitle: "bazz",
+            posterPath: "fizz",
+            releaseDate: "foo",
+            revenue: 1
+        )
+        view = DetailMovieMockView()
+        movieAPIService = MockNetworkService(movie: MovieDetail(
+            backdropPath: "foo",
+            budget: 1,
+            id: 1,
+            overview: "bazz",
+            originalTitle: "fizz",
+            posterPath: "bazz",
+            releaseDate: "foo",
+            revenue: 1
+        ))
+
+        XCTAssertNil(presenter)
```

Comment:
```suggestion
        XCTAssertNil(presenter)
```
в 1 методе 1 тест пишем